templates/base.html

```diff
@@ -7,6 +7,7 @@
     <link rel="stylesheet" type="text/css" href="/static/css/base.css">
     <link rel="stylesheet" type="text/css" href="/static/css/home.css">
     <link rel="stylesheet" type="text/css" href="/static/css/authentication.css">
+    <link rel="stylesheet" type="text/css" href="/static/css/search.css">
```

Comment:
I was thinking about adding template-specific css links to `{% block head %}`  (direct in html file related with styles , not in `base.html`). But maybe @etude-ist will determine it.

Reply:
Yeah it seems more reusable and less 'garbage' making.